bciers/apps/reporting/src/app/bceidbusiness/industry_user_admin/reports/[version_id]/FormContext.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {
+  createContext,
+  ReactElement,
+  useContext,
+  useState,
+} from "react";
+import { TaskListElement } from "@bciers/components/navigation/reportingTaskList/types";
+import {
+  operationReviewSchema,
+  operationReviewUiSchema,
+} from "@reporting/src/data/jsonSchema/operations";
+import { RJSFSchema } from "@rjsf/utils";
+
+const taskListElements: TaskListElement[] = [
+  {
+    type: "Section",
+    title: "Operation information",
+    isExpanded: true,
+    elements: [
+      { type: "Page", title: "Review Operation information", isChecked: true },
+      { type: "Page", title: "Person responsible", isActive: true },
+      { type: "Page", title: "Review facilities" },
+    ],
+  },
+];
+
+export const FormContext = createContext<{
+  taskList: TaskListElement[];
+  setTaskList: (taskList: TaskListElement[]) => void;
+  formData: any;
+  setFormData: (formData: FormData) => void;
+  formSchema: RJSFSchema;
+  setFormSchema: (formData: RJSFSchema) => void;
+  formUiSchema: RJSFSchema;
+  setFormUiSchema: (formUiSchema: RJSFSchema) => void;
+  formSubmitHandler: () => void;
+  setFormSubmitHandler: (formSubmitHandler: () => void) => void;
```

Comment:
Perhaps there is some complexity I'm missing with your forms though I recommend talking to @shon-button about how to share state in between pages the NextJS way using app router and dynamic routes instead of using React context.